src/routes/applications.ts

```diff
@@ -64,8 +64,7 @@ export const downloadImageConfig: RequestHandler = async (req, res) => {
 		res.json(config);
 	} catch (err) {
 		if (err instanceof UnauthorizedError) {
-			res.status(404).send(err.message);
-			return;
+			err = new BadRequestError(err);
```

Comment:
Why change the response? Shouldn't this be NotFoundError instead?

Reply:
oh ffs yes

Reply:
fixed